<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <!--  param to change: 'src_bag_path', 'dst_bag_path', 'imu_filename', 'image_timestamp_filename', 'image_dir'  -->
    <node pkg="rosbag_data_inserter" type="imu_camera_frame_inserter" name="imu_camera_frame_inserter_node"
          output="screen">
        <param name="src_bag_path" value="/home/csl/dataset/data_416/data_2023416164855/lidar/lidar.bag"
               type="string"/>
        <param name="dst_bag_path" value="/home/csl/dataset/data_416/data_2023416164855/lidar_imu_camera.bag"
               type="string"/>

        <param name="imu_filename" value="/home/csl/dataset/data_416/data_2023416164855/imu_adis_new.txt"
               type="string"/>

        <param name="image_timestamp_filename"
               value="/home/csl/dataset/data_416/data_2023416164855/stamp_rearrange_cam_rs.txt" type="string"/>
        <param name="image_dir" value="/home/csl/dataset/data_416/data_2023416164855/image_rearrange/cam_rs"
               type="string"/>

        <param name="lidar_src_topic" value="/velodyne_points" type="string"/>
        <param name="lidar_dst_topic" value="/lidar_raw_frames" type="string"/>
        <param name="lidar_dst_frame_id" value="lidar" type="string"/>

        <param name="imu_dst_topic" value="/imu_raw_frames" type="string"/>
        <param name="imu_dst_frame_id" value="imu" type="string"/>

        <param name="camera_dst_topic" value="/cam_rs_raw_frames" type="string"/>
        <param name="camera_dst_frame_id" value="cam_rs" type="string"/>

        <param name="gps_week" value="2258" type="int"/>

    </node>
</launch>